<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brick Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>370</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="background">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>641</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <widget class="QPushButton" name="tetrisButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>158</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Tetris</string>
       </property>
      </widget>
      <widget class="QPushButton" name="snakeButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>158</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Snake</string>
       </property>
      </widget>
      <widget class="QPushButton" name="racingButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>158</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Car racing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>161</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Classic tank</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>467</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
